packages/aws-cdk/lib/init.ts

```diff
@@ -1,3 +1,4 @@
+import { JsiiPacmak } from '@aws-cdk/jsii-pacmak-version';
```

Comment:
Don't believe this is correct (or if it is, only incidentally correct). This is going to be the version of `jsii-pacmak-version` at the point the CLI was released, which is not necessarily going to be the version the CDK framework libraries in the current project are going to be built against.

And if your argument is, a CLI will always generate a project with framework versions equal to itself (fair enough), then we don't actually need the additional package. The CLI can then "at build time" check the version of jsii-pacmak directly and store it in itself.

That saves us an additional package we would want to get rid of in the future anyway.

Reply:
You (and @eladb) are right - this is a little heavy-handed. I've moved that information closer to it's usage location - now including that data into "build-info.json", as this can be useful information in other contexts, too (especially when porting artifacts to different versioning systems, etc...).